src/main/scala/exercises/06_Collections.scala

```diff
@@ -65,7 +65,10 @@ object Collections {
    *  - ... etc
    */
   def nthElementInList[T](n: Int, l: List[T]): T = {
-    error("fix me")
```

Comment:
I tried to do something similar, but couldn't, so thank you for figuring it out! What I tried to do was to remove all the elements in indices 0 through n-1, and then returning the head of the remaining list.